daisi/src/cpps/amr/physical/amr_physical_asset.cpp

```diff
@@ -28,12 +28,12 @@ namespace daisi::cpps {
 static constexpr uint32_t kUpdateFrequencyHz = 10;
 
 AmrPhysicalAsset::AmrPhysicalAsset(AmrAssetConnector connector, const Topology &topology)
-    : connector_(std::move(connector)) {
+    : connector_(std::move(connector)), fsm(TaskExecutionState::kFinished) {
```

Comment:
I would prefer to add a comment for the constructor.

It would be possible to use `kInitial` to signal that there is no previeous task but this would only add an unnecessary duplicate of the `kFinished` state in the fsm. (I would not like `kIdle` because a task does not progress idly)